source/geod24/concurrency.d

```diff
@@ -2230,6 +2316,19 @@ final public class Channel (T) : Selectable
             if (!this.select_state || this.select_state.tryConsume(this.sel_id, this.success))
                 this.blocker.notify();
         }
+
+
+        /***********************************************************************
+
+            Terminate ChannelQueueEntry so that it is neutralized
+
+        ***********************************************************************/
+
+        void release() nothrow
+        {
+            if (this.blocker.stopTimer())
+                this.pVal = cast(T*) 0x0; // Sanitize pVal so that we can catch illegal accesses
```

Comment:
I think this can just be `= null;`?

Reply:
The cast will make the code non-`@safe`. It would be good to make most of the code we write `@safe`. But we can do it later too..